frameworks/kafka/cli/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+
+CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $CUR_DIR
+
+EXE_NAME="dcos-service-cli"
+
+# required env:
+export REPO_ROOT_DIR=$(dirname $(dirname $(dirname $CUR_DIR)))
+export REPO_NAME=$(basename $REPO_ROOT_DIR)
+$REPO_ROOT_DIR/tools/build_go_exe.sh frameworks/kafka/cli/ ${EXE_NAME}-linux linux
+$REPO_ROOT_DIR/tools/build_go_exe.sh frameworks/kafka/cli/ ${EXE_NAME}-darwin darwin
+$REPO_ROOT_DIR/tools/build_go_exe.sh frameworks/kafka/cli/ ${EXE_NAME}.exe windows
+
+echo $(pwd)/${EXE_NAME}-linux $(pwd)/${EXE_NAME}-darwin $(pwd)/${EXE_NAME}.exe
```

Comment:
What is the purpose of this final echo? At first I thought it was used to capture the artefact paths, but that seems to be hard-coded dependent on the `CLI_DIR`.